websites/api-docs-smoke-test/src/@commercetools-docs/gatsby-theme-docs/overrides/site-icon.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Icons } from '@commercetools-docs/gatsby-theme-docs';
```

Comment:
This looks weird. The icons are shipped with the docs theme but then we override the theme using the icons exported from the theme.

There are 2 ways of doing this:
* keep the icons in the theme, do not expose them, use a config option to tell the theme which icon type to render.
* do not keep the icons in the theme, just use the override placeholder. Consumers of the theme should then provide the icon.

For me it seems like option 1 is the preferred approach.

Reply:
hi @emmenko, I did go through the requirements with timon before we chose this approach and we had the other two you mentioned on the list, too. 
 
 * theme or ui kit should provide the typically used icons to sites centrally, so the frequently used "main" ones can be updated centrally
* a site needs to use an icon as its home icon but also anywhere else in the content (not only the "own" icon, also for referring to other sites). So some package needs to export them; alternatively they would have to be available in any MDX by default. 
 * the icons are not a 1:1 mapping to the color presets, so binding it to the site color does not work
 * avoid introducing a third way to customize.  We have overrides and we have config flags in the theme config already. 
 * a site should be able to provide an own, custom icon without having to have it accepted into the docs-kit. It's not a use case already now but our goal is to have many more and less prominent pieces of commercetools docs use the kit and it would be slow and inflexible to force all icons into the kit.  (sure, this part is a bit opinionated)

What I would consider improving here is to rather make the icons a part of the docs/ui-kit  and not the theme.